README.md

```diff
@@ -176,36 +176,96 @@ Parameters:
 - type: String
 - example: 659bbda98db7a017ddd9147a
 - description: ADP application id to be updated. It can be obtained from ADP app version page- it is available in url
-4. imagename
+4. appVersion
 - type: String
--  example: ExampleApp
-- description: Image name which should be used while uploading container image to ADP
+- example: 1
+- description: ADP application version which needs to be downloaded.
 5. github_repo
 - type: String
 - example: https://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@github.com/aruba/aruba-iotops-example-ble
+- description: Git repo url detail, where updated json can be pushed
+6. app_bundle_path
```

Comment:
This repo is just an example, not template. I think we can directly use aruba-iotops-example-ble/resource/appbundle/appbundle.json without providing the flexibility of customizing it, for users' better understanding.

Reply:
readme updated for removal of extra jenkins pipeline